src/Core/Grid/Column/Type/Common/PositionColumn.php

```diff
@@ -53,25 +53,25 @@ public function getType()
      */
     protected function configureOptions(OptionsResolver $resolver)
     {
+        parent::configureOptions($resolver);
+
         $resolver
             ->setRequired([
                 'id_field',
                 'position_field',
                 'update_route',
             ])
             ->setDefaults([
-                'sortable' => true,
```

Comment:
removed sortable in common columns? Was it intentional?

Reply:
Have no ideas now, why Sarjon did it => let's figure out ourselves

Reply:
All the info of why I know is here while rebasing:
![image](https://user-images.githubusercontent.com/32385991/73633828-4e27dd80-4668-11ea-9b17-ec987b6e52d9.png)

Reply:
Might be a mistake. Better leave it here, removing it could even be seen as a Breaking Change

Reply:
Oh, just figured it out - it is a refactoring, see parent::configureOptions($resolver); 
https://github.com/tdavidsonas88/PrestaShop/blob/5d38d8cd97a55e6614a48eb04453e691baef1b06/src/Core/Grid/Column/AbstractColumn.php#L112-L125

Reply:
OK, got it 👍